taigit/src/components/GitHub.js

```diff
@@ -1,23 +1,24 @@
 import React, { Component } from 'react'
 import BarChart from './charts/BarChart';
+import NumberDisplay from './NumberDisplay'
+import barChartData from './charts/barChartData'
 
 export default class GitHub extends Component {
   render() {
     return(
       <div className="app-page">
         <h2>GitHub</h2>
-        <div className="chart chart-horizontal-primary">
-        <BarChart/>       
-        </div>
-        <div className="chart chart-square-primary"></div>
-        <div className="chart chart-horizontal-primary"></div>
-        <div className="chart chart-square-primary"></div>
-        <div className="chart chart-horizontal-primary"></div>
-        <div className="chart chart-square-primary"></div>
-        <div className="chart chart-square-primary"></div>
-        <div className="chart chart-horizontal-primary"></div>
-        <div className="chart chart-square-primary">
-        </div>
+
+          <div className="chart chart-bar">
+              <span className="chart-title">Commits Per Member</span>
+              <BarChart chartData={barChartData}/>
+          </div>
+
+          <div className="pull-req-stats">
```

Comment:
The NumberDisplay components can be taken out of this div, as there aren't any styles being computed on the "pull-req-stats" div.